samza-core/src/main/java/org/apache/samza/storage/StorageManagerUtil.java

```diff
@@ -115,6 +115,16 @@ public static boolean isOffsetFileValid(File storeDir, String offsetFileName) {
     return hasValidOffsetFile;
   }
 
+  /**
+   * Check if the storeDirectory exists.
```

Comment:
I am not sure if the current check needs a separate method by itself; especially under storage util. 
We use this to determine only if the store exists. Maybe we can rename this method to `storeExists` and also check for non-zero file count for now? 

I am also fine with just the existence of the directory check. Although, i feel we can inline that as opposed to adding a new method in the storage util.

Thoughts?

Reply:
Done, 
yup, makes sense to rename to storeExists and add non-zero file count.